testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/api.tests.js

```diff
@@ -7,6 +7,8 @@ import { isObject } from 'core/utils/type';
 const { test, module: testModule } = QUnit;
 
 const TOOLBAR_FORMAT_WIDGET_CLASS = 'dx-htmleditor-toolbar-format';
+const DISABLED_STATE_CLASS = 'dx-state-disabled';
+const DX_BUTTON_CLASS = 'dx-button';
```

Comment:
I don't think there can be some "not dx" button class)
```suggestion
const BUTTON_CLASS = 'dx-button';
```